### ISSUE TITLE
feature request: add the `standalone` option to generate certs

### ISSUE THREAD

**Issue by vpistis** [OpenMediaVault-Plugin-Developers/openmediavault-letsencrypt#10]
Hi,

this plugin works very well for me, but in some case I must run a `standalone` webserver to renew certs. There is some chance to add this option to the plugin?

Thanx a lot

**Comment by ryecoaaron** [comment]
I don't see much of a reason to add this since you are running it from a web server.  If you don't want to use the OMV site itself, you could install the nginx plugin and create a server just for the cert.

**Comment by vpistis** [comment]
you're right, in fact, thinking about it well,  what I would like is the possibility to use the `nginx` plugin because ports http are closed into my server.
I already use OMV nginx plugin (and I'm enthusiast :) ) and I have Nextcloud only in https, this letsencrypt plugin don't correctly renew the certificates due to the block on 80 port for nextcloud...OMV is already served only in https.

For me is a great feature of this plugin that permits to add automatically a letsencrypt certificate into respective section of OMV without manually copying cert files.

At this moment the only solution that I see is to open http ports.

thanx
